nginx.conf

```diff
@@ -1,20 +1,18 @@
-worker_processes ${{NUM_WORKERS}};
-error_log stderr notice;
-daemon ${{USE_DAEMON}};
-pid logs/nginx.pid;
 env PORT;
 env SSL_PORT;
 env SSL_CERT_NAME;
-env DATABASE_USERNAME;
-env DATABASE_URL;
-env DATABASE_PASSWORD;
```

Comment:
Why are we removing these? :thinking:

Reply:
Feel free to correct me, but my understanding is that the `env` declaration only applies to what nginx should be reading. There's no need for nginx to read the db configuration b/c that's a handled in lapis. (And that means I should also remove `MAIL_`* ones.)

Reply:
I've just confirmed you do need to whitelist every env var you want to be able to access from within OpenResty, and that means all of them. I'm reverting just these changes.